include/silo/storage/sequence_store.h

```diff
@@ -39,7 +39,7 @@ struct SequenceStoreInfo {
 
 class SequenceStore {
   private:
-   unsigned sequence_count;
+   unsigned sequence_count{};
```

Comment:
Personally, I am not happy to initialize the `sequence_count` here, but since we do not have a real constructor, I think it is fine.

Reply:
Left it as it is since there is no real constructor atm.